manifest.yaml

```diff
@@ -36,7 +37,6 @@ extensions:
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/extension/storage/filestorage v0.95.0
 processors:
   - gomod: go.opentelemetry.io/collector/processor/batchprocessor v0.95.0
-  - gomod: go.opentelemetry.io/collector/processor/memorylimiterprocessor v0.95.0
```

Comment:
Why are you removing the processor?

Reply:
Reverted also, you're right, make no sense to remove it. not for now.